code/game/machinery/vending/vendor_types/crew/synthetic.dm

```diff
@@ -28,6 +29,7 @@
 		list("Toxin Firstaid Kit", 6, /obj/item/storage/firstaid/toxin, null, VENDOR_ITEM_REGULAR),
 		list("Oxygen Firstaid Kit", 6, /obj/item/storage/firstaid/o2, null, VENDOR_ITEM_REGULAR),
 		list("Radiation Firstaid Kit", 6, /obj/item/storage/firstaid/rad, null, VENDOR_ITEM_REGULAR),
+		list("Empty Adv Firstaid Kit", 1, /obj/item/storage/firstaid/adv/empty, null, VENDOR_ITEM_REGULAR),
```

Comment:
I don't really like that we've started handing these out empty. Let's take this out.